### PR TITLE
Setup command package

### LINKED ISSUE #36: set up Command class

### PR DESCRIPTION
**Pull request by warrencxw**
Add abstract run method to Command class to run the command.

Fixes #36

### CODE REVIEW

File: src/main/java/seedu/duke/command/Command.java [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
 public abstract class Command {
+    /**
+     * Runs the command with the information parsed from the user input, using the specified
+     * Profile and TextUI objects.
+     * @param ui      A user interface to which the command will read its input from and print its output to.
+     * @param profile A Profile object from which Session, Activity and other objects are used to run
+     *                the command.
+     */
+    public abstract void run(TextUI ui, Profile profile);
 }
```
Comment (matheril) [inline src/main/java/seedu/duke/command/Command.java:20]: Minor change but consider adding a newline to the end.
Looks good to merge otherwise

**Review by matheril — CHANGES_REQUESTED** [review]
Consider adding a newline to the end

File: src/main/java/seedu/duke/command/Command.java (hunk 1 above)
Comment (warrencxw) [reply src/main/java/seedu/duke/command/Command.java]: Added newline in next commit.

**Review by matheril — APPROVED** [review]
Looks good to merge

**Review by froststein — APPROVED** [review]
Looks good for merge

**Pull request merged**